src/copy.rs

```diff
@@ -285,6 +286,12 @@ impl<'a> Options<'a> {
     pub fn copy(self, source: Source<'_>, mime_type: MimeType) -> Result<(), Error> {
         copy(self, source, mime_type)
     }
+
+    /// Invokes the copy_multi operation. See `copy_multi()`.
+    #[inline]
+    pub fn copy_multi(self, sources: Vec<(Source<'_>, MimeType)>) -> Result<(), Error> {
```

Comment:
I'm not a big fan of this tuple; I think I'd make it its own struct with two named fields.

Reply:
Fixed.